_posts/2020-01-01-2019-summary.md

```diff
@@ -227,5 +227,4 @@ These are some of the projects that are currently in progress and are likely to
 * A new [Parquet writer]({{site.github_repo_url}}/pull/2004) optimized to work efficiently 
   within Presto.
 
-... and many, many more. Follow us on [Twitter](https://twitter.com/{{site.twitter_user}}) and 
-[join our Slack](/slack.html).
+... and many, many more.
```

Comment:
Can we get rid of this, too? There's almost never a right time to say "and more," and we already say "some of the projects," up top, implying this list is non-exhaustive.

Reply:
Thats a VERY old blog post .. not going to bother.